README.md

```diff
@@ -81,8 +100,7 @@ Describe the differences between unit and functional testing. What type of testi
 
 Your answer:
 ```text
-
-Replace this with your answer
+Unit tests are testing one particular element of the functionality as you go along. For example, testing using a puts statement to return the value in Pry. Functional testing are in-depth testing that confirms that the system does what it is expected to, and that something that was once working is not being broken by new code that has been added. RSpec is functional testing as it tests many different elements of the functionality at once and returns whether each test was a success or failure.
```

Comment:
Not quite. RSpec is actually unit testing, and checks whether a specific method has a specific output (testing individual methods)